lib/sqlitex/row.ex

```diff
@@ -22,10 +22,21 @@ defmodule Sqlitex.Row do
     nil
   end
 
+  # date is empty ""
+  defp translate_value({date, "date"}) when date=="", do: to_date(date)
+
   defp translate_value({date, "date"}) when is_binary(date), do: to_date(date)
 
+  # time is empty ""
+  defp translate_value({time, "time"}) when time=="", do: to_time(time)
+
   defp translate_value({time, "time"}) when is_binary(time), do: to_time(time)
 
+  # datetime is empty ""
+  defp translate_value({datetime, "datetime"}) when datetime=="" do
```

Comment:
It's probably nicer to use pattern matching than a guard clause:`defp translate_value({"", "datetime"}) do`